apisix/cli/ops.lua

```diff
@@ -728,14 +729,16 @@ local function start(env, ...)
     local pid = util.read_file(pid_path)
     pid = tonumber(pid)
     if pid then
-        local lsof_cmd = "lsof -p " .. pid
-        local res, err = util.execute_cmd(lsof_cmd)
-        if not (res and res == "") then
-            if not res then
-                print(err)
-            else
-                print("APISIX is running...")
-            end
+        local signone = 0
```

Comment:
Better to add a test case to cover the changes. You can find some references from the `t/cli/` directory.

Reply:
Added test cases to cover in `t/cli/test_ops.sh`.